app/views/user/_exercises_table.erb

```diff
@@ -18,7 +18,7 @@
       <th>Language</th>
       <th>Exercise</th>
       <th>Iterations</th>
-      <th data-sorted="true" data-sorted-direction="descending">Latest Iteration</th>
+      <th data-sorted="true" data-sorted-direction="ascending">Latest Iteration</th>
```

Comment:
This is where the exercises need to be sorted:
https://github.com/exercism/exercism.io/blob/master/app/views/user/_exercises_table.erb#L26